docs/pkglists/Fedora_29.x86_64

```diff
@@ -8,23 +8,23 @@ debug/x86_64/bareos-database-tools-debuginfo
 debug/x86_64/bareos-debuginfo
```

Comment:
I belive we're not building Fedora 29 anymore. So maybe we could remove pkglist?